kubejs/server_scripts/_packmode.js

```diff
@@ -11,18 +11,34 @@ if (!config || !config.mode) {
     console.log(`Created new ${configName}`);
     config = defaultConfig;
 }
-if (config.mode == 'none') {
+if (config.mode === 'none') {
     JsonIO.write(configName, defaultConfig);
     config.mode = defaultConfig.mode;
-    console.log(`Overwrote ${configName}, because the mode 'none' was found. Valid modes are 'normal', 'hardcore' and 'omega'.`);
+    console.log(`Overwrote ${configName}, because the mode 'none' was found. Valid modes are 'normal', 'expert' and 'omega'.`);
 }
 
 let packMode = config.mode;
 
 global.packmode = packMode;
 global.isNormalMode = packMode == 'normal';
-global.isHardcoreMode = packMode == 'hardcore';
+global.isExpertMode = (packMode == 'expert') || global.isOmegaMode;
 global.isOmegaMode = packMode == 'omega';
 const isNormalMode = packMode == 'normal';
-const isHardcoreMode = packMode == 'hardcore';
-const isOmegaMode = packMode == 'omega';
\ No newline at end of file
+const isExpertMode = (packMode == 'expert') || isOmegaMode;
```

Comment:
References `isOmegaMode` before defining it, likely you want to swap the lines (maybe reverse all three so it is consistent.

Ditto for `global.isExpertMode` above

Reply:
this is js we can get away with this, although i don't think i cant get away with this on expert mode since it does some internal conversion

Reply:
Oh BTW we won't need an "isOmegaMode" thing, because Omega mode will be a different pack in a different repo when the time comes (too many changes).